static/app/views/performance/database/useAvailableDurationAggregates.tsx

```diff
@@ -1,50 +1,26 @@
-import {decodeScalar} from 'sentry/utils/queryString';
-import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import {DEFAULT_DURATION_AGGREGATE} from 'sentry/views/performance/database/settings';
 
 // TODO: Type more strictly, these should be limited to only valid aggregate
 // functions
-type Query = {
-  aggregate: string;
-};
-
-type Result = {
-  availableAggregates: string[];
-  selectedAggregate: string;
-};
+type Result = string[];
 
 export function useAvailableDurationAggregates(): Result {
   const organization = useOrganization();
-  const location = useLocation<Query>();
 
-  let availableAggregates = ['avg'];
+  let availableAggregates: string[] = [];
 
-  const arePercentilesEnabled = organization.features?.includes(
-    'performance-database-view-percentiles'
-  );
+  availableAggregates.push(DEFAULT_DURATION_AGGREGATE);
 
-  if (arePercentilesEnabled) {
+  if (organization.features?.includes('performance-database-view-percentiles')) {
     availableAggregates = [...availableAggregates, ...['p50', 'p75', 'p95', 'p99']];
   }
 
   // TODO: Enable this on the backend
-  const isMaxEnabled = false;
-  if (isMaxEnabled) {
+  const areLimitsEnabled = false;
+  if (areLimitsEnabled) {
```

Comment:
l: Given we're here and still not changing this 2nd time around, can we remove?

Reply:
😂 I keep trying to sneak this in 'cause I'm working on it right now. I'll remove!